test/KnormRelations.spec.js

```diff
@@ -897,6 +909,8 @@ describe('KnormRelations', () => {
             );
 
             await Image.delete({ where: { id: 2 } });
+            await Image.delete({ where: { id: 3 } });
```

Comment:
we can batch the image deletion as well: `await Image.delete({ where: Image.where.in({ id: [2, 3] }) })`